src/js/models/appUser.js

```diff
@@ -18,5 +20,28 @@ module.exports = BaseModel.extend({
             properties: {},
             conversationStarted: false
         };
+    },
+
+    isDirty: function() {
+        var hasChanged = false;
+
+        _.each(AppUserModel.EDITABLE_PROPERTIES, function(property) {
+            hasChanged = hasChanged || this.hasChanged(property);
+        }.bind(this));
+
+        return hasChanged;
+    },
+
+    saveIfDirty: function() {
+        if (this.isDirty()) {
+            return this.save();
+        } else {
+            return $.Deferred().resolve().promise();
```

Comment:
that promise should replicate what  `this.save()` is return (I think it passes the model as the first parameter)